test-utils/tests/test-pushStateMock.js

```diff
@@ -167,14 +167,14 @@ o.spec("pushStateMock", function() {
 		})
 	})
 	o.spec("set protocol", function() {
-		o("setting protocol throws", function(done) {
-			var old = $window.location.href
+		o("setting protocol throws", function() {
 			try {
 				$window.location.protocol = "https://"
 			}
 			catch (e) {
-				done()
+				return
```

Comment:
Wonder if `o.throws(func)` should be added to ospec? I see patterns like this enough in the tests that I feel such an addition could be merited.